resources/views/site/help/import.blade.php

```diff
@@ -0,0 +1,94 @@
+
+    <hr class="mb-4" />
+
+    <p class="text-center font-weight-bold">How to get your export data from Instagram:</p>
+    <ol class="pb-4">
+        <li class="mb-2">
+            <span>Follow the Instagram instructions on <strong>Downloading a copy of your data on Instagram</strong> on <a href="https://help.instagram.com/181231772500920" class="font-weight-bold">this page</a>. <strong class="text-danger small font-weight-bold">Make sure you select the JSON format</strong></span>
+        </li>
+        <li class="mb-2">
+            <span>Wait for the email from Instagram with your download link</span>
+        </li>
+        <li class="mb-2">
+            <span>Download your .zip export from Instagram</span>
+        </li>
+        <li class="mb-2">
+            <span>Navigate to the <a href="/settings/import" class="font-weight-bold">Import</a> settings page</span>
+        </li>
+        <li class="">
+            <span>Follow the instructions and import your posts 🥳</span>
+        </li>
+    </ol>
+    <hr class="mb-4" />
+
+    <p class="text-center font-weight-bold">Import Limits</p>
+
+    <div class="list-group pb-4">
+        <div class="list-group-item d-flex justify-content-between align-items-center">
+            <div>
+                <p class="font-weight-bold mb-0">Max Posts</p>
+                <p class="small mb-0">The maximum imported posts allowed</p>
+            </div>
+            <div class="font-weight-bold">{{ config('import.instagram.limits.max_posts') == -1 ? 'Unlimited' : config('import.instagram.limits.max_posts') }}</div>
+        </div>
+        <div class="list-group-item d-flex justify-content-between align-items-center">
+            <div>
+                <p class="font-weight-bold mb-0">Max Attempts</p>
+                <p class="small mb-0">The maximum import attempts allowed<br />(counted as total imports grouped by day)</p>
+            </div>
+            <div class="font-weight-bold">{{ config('import.instagram.limits.max_attempts') == -1 ? 'Unlimited' : config('import.instagram.limits.max_attempts') }}</div>
+        </div>
+        <div class="list-group-item d-flex justify-content-between align-items-center">
+            <div>
+                <p class="font-weight-bold mb-0">Video Imports</p>
+                <p class="small mb-0">The server supports importing video posts</p>
+            </div>
+            <div class="font-weight-bold">{{ config('import.instagram.allow_video_posts') ? '✅' : '❌' }}</div>
+        </div>
+    </div>
+
+    <hr class="mb-4" />
+
+    <p class="text-center font-weight-bold mb-0">Import Permissions</p>
+    <p class="text-center small">Who is allowed to use the Import feature</p>
+
+    <div class="list-group">
```

Comment:
Is it possible to have no limits at all? e.g., all accounts can perform imports? If so, it might be worth looking at copy that handles that "Anyone can use Import on this server", rather than listing the individual potential rules.